### DISCUSSION TITLE
大佬lobechat的插件功能是用的官方的functioncalling吗

### DISCUSSION THREAD

**Discussion by dao-zero** [lobehub/lobehub#1243] [General Question] [answered]
如题，比较好奇lobechat是借助functioncalling功能实现的插件调用还是完全重新实现了一个插件，因为官方的插件再问数学题的时候可以自己编程解决然后返回结果，所以好奇lobechat是不是也是function calling

**Comment by arvinxx** [comment, marked as answer]
是 function calling
